hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -433,7 +433,7 @@ private boolean initializeFromFilesystem(String initializationTime, List<Metadat
             }
             ValidationUtils.checkState(functionalIndexPartitionsToInit.size() == 1, "Only one functional index at a time is supported for now");
             partitionName = functionalIndexPartitionsToInit.iterator().next();
-            fileGroupCountAndRecordsPair = initializeFunctionalIndexPartition(partitionName);
+            fileGroupCountAndRecordsPair = initializeFunctionalIndexPartition(partitionName, commitTimeForPartition);
```

Comment:
Be consistent on using `commit` or `instant`: `commitTimeForPartition` and `instantTime`.